bionic/core/task_execution.py

```diff
@@ -620,3 +570,51 @@ def _load_value_hashes(self):
                 )
             result_value_hashes_by_dnode[accessor.query.dnode] = value_hash
         self._result_value_hashes_by_dnode = result_value_hashes_by_dnode
+
+    def strip_state_for_subprocess(self, new_task_states_by_key=None):
```

Comment:
Yeah... I first moved this to `Entry` because we were sending `Entry.compute` to the subprocesses with the second commit. But in the third commit, I started sending `TaskCompletionRunner` which takes `TaskState`s, and I had to move the method back.